src/components/__snapshots__/Answer.spec.tsx.snap

```diff
@@ -38,7 +38,7 @@ exports[`Answer matches snapshot 1`] = `
       <div
         className="answer-answer"
       >
-        <div
+        <span
```

Comment:
shouldn't these stay as block elements?

Reply:
Tutor defaults block to false so when building out the `Content` component, I mirrored that: https://github.com/openstax/tutor-js/blob/assignment-component-library/shared/src/components/html.js#L27. I probably should update the specs to have it on by default instead, which matches the Tutor specs as well, I'll do that and add one for the false case.

Reply:
Looking at this again, I'm okay with the snapshot change because it matches how we'll be rendering it. The Tutor specs were overriding the html component to return divs, even though the real render case is inline.

Reply:
hmm so we'll have `<ol>`s inside of `<span>`s? I just wasn't sure about that html.

Reply:
I don't think we'll see that in practicality, maybe I should update the specs with a better default case for that content. The `ol` content comes from a tutor spec, I don't think I've ever seen any real content with block elements.